client/dashboard/profile-wizard/steps/start/index.js

```diff
@@ -128,8 +168,28 @@ class Start extends Component {
 		);
 	}
 
+	getBenefits() {
+		const activatedPlugins = get( wcSettings, [ 'onboarding', 'activePlugins' ], [] );
+		if ( activatedPlugins.includes( 'jetpack' ) ) {
+			return filter( benefits, benefit => benefit.plugins.includes( 'woocommerce-services' ) );
+		}
+		return filter( benefits, benefit => benefit.plugins.includes( 'jetpack' ) );
```

Comment:
> Might be misunderstanding the use here, but I would think it should return all benefits if both are deactivated.

The design for needing to install both plugins shows just 4 benefits (security, sales tax, speed, mobile app), and not the full set of 6 benefits. I originally thought this was to reduce the amount of information and features we are throwing in front of their face.

I might be wrong about that, and maybe it was an oversight when branching the designs.

Jay's on vacation this week so we should probably just pick showing 4 or 6. Do you think we should show all of them?

I agree the logic here can be made a bit clearer though either way.

Reply:
Cleaned up the logic in https://github.com/woocommerce/woocommerce-admin/pull/2825/commits/cf79b019e0d16b50c4c31da6e702ac7caf9bf9d7, to go with the `visible` key. I'm still just showing the four benefits, but it's a bit clearer what's going on now IMO.

Reply:
Logic looks great!  I like the idea of showing all the benefits, but maybe it's a too much information as you noted or an "above the fold" situation.  We can leave it as is unless Jay says otherwise.